### PR TITLE
UI Fix - Input get OFF focus on focus

### PR DESCRIPTION
**Pull request by rahul3v**
Input get OFF focus on focus + few tweaks

### **Now**
inputFieldAutofocusError

-------------------------------

### **After change**

AfterinputFieldAutofocusError

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Lightdash is a BI tool that is fully integrated with your dbt project, allowing
 * [x] 👩‍💻 Declare dimensions and metrics in yaml alongside your dbt project
 * [x] 🤖 Automatically creates dimensions from your dbt models
 * [x] 📖 All dbt descriptions synced for your users
-* [x] 📊 Simple data visualisations for your metrics
+* [x] 📊 Simple data visualizations for your metrics
```
Comment (owlas) [inline README.md:38]: Still need to write a contributors guide but we use US English in the code base. so *-ize* is preferred.

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,7 @@ Recent and upcoming changes to lightdash
  - Stick sidebar on top left 
  - The stuck problem with scrollbar that shows only limited content on overflow
  - Errors in `meta` tags (such as invalid references to other dimensions) no longer crash the server and errors are shown in the UI
+ - UI fixes - Error in input focus
```
Comment (owlas) [inline CHANGELOG.md:17]: Again, contribution style, this should be a very user friendly message (not for developers).

Perhaps: "fix in the UI where the search box would be cutoff"

**Review by owlas — CHANGES_REQUESTED** [review]

**Comment by owlas** [comment]
@all-contributors add @rahul3v for design, code, docs, bugs

**Comment by owlas** [comment]
Thanks @rahul3v left a quick review. I've just added #138 to update the contributors guide, to help in future

**Comment by owlas** [comment]
@rahul3v please could you:

1. revert the British spelling (visualisations -> visualizations)
2. squash and rebase your commits

**Comment by owlas** [comment]
Thanks @rahul3v just need to rebase on main so that the changelog is up to date:

```
git fetch -a
git rebase origin/main
git push -f
```

**Comment by rahul3v** [comment]
@owlas `rebase` cmds done :+1:

**Comment by owlas** [comment]
Hey @rahul3v thanks so much. It still looks like the changelog doesn't have the `0.2.7` entry which is on main, can you try again? Not sure what happened there

(although GitHub is offering to auto merge, I'm not sure the entry will go in the right section)

**Comment by owlas** [comment]
Hey @rahul3v you've got a merge commit now. There should be just a single commit here rebased on `main`.

IF you do: `git rebase -i main` you can squash those two commits together. 

I can do this for you if that helps

**Comment by rahul3v** [comment]
Well @owlas, You should try at your end, I think  my `main` branch was not updated for this repo, leads to some issue, but still it's I mange to get all the updates from the root repo, :)

It would be batter if you simplify the steps you follow, for the next time, how you do that regarding `rebase`, bit fuzz , I follow the steps but still some bugs I feel 

Can I add new features into it, If yes where to discuses about it

**Comment by owlas** [comment]
No prob: 

1. Could you please follow these instructions to allow us to edit your fork: https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork
2. If you have any issues with the contribution workflow you can definitely open an issue or add to #138
3. If you want a more open ended discussion with the devs, then please drop a new topic in our forum! https://community.lightdash.com

**Comment by rahul3v** [comment]
The edit is allowed , and everything looks good, you can merge the commit

**Comment by owlas** [comment]
@rahul3v finished!

You can see that now: 
* There is just one commit (which is directly on top of `main`)
* The changelog contains a new entry in the "unreleased" section - not in a tagged release.
* There is no merge commit

I did this by:
1. Cloning your fork
2. Creating a reference to the main lightdash repo `git remote add upstream git@github.com:lightdash/lightdash.git`
3. Updating `main` to latest `git pull upstream`
4. Rebasing and cleaning commits `git rebase -i upstream/main`
5. Pushing changes `git push -f`

**Review by owlas — APPROVED** [review]

**Pull request merged**